.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,37 +1,20 @@
-<!---
-☝️ PR title should follow conventional commits (https://conventionalcommits.org)
-
-Please carefully read the contribution docs before creating a pull request
- 👉 https://nuxt.com/docs/community/contribution
--->
-
 ### 🔗 Linked issue
 
 <!-- Please ensure there is an open issue and mention its number as #123 -->
 
-### ❓ Type of change
-
-<!-- What types of changes does your code introduce? Put an `x` in all the boxes that apply. -->
-
-- [ ] 📖 Documentation (updates to the documentation, readme or JSdoc annotations)
-- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
-- [ ] 👌 Enhancement (improving an existing functionality like performance)
-- [ ] ✨ New feature (a non-breaking change that adds functionality)
-- [ ] 🧹 Chore (updates to the build process or auxiliary tools and libraries)
-- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)
-
 ### 📚 Description
 
-<!-- Describe your changes in detail -->
-<!-- Why is this change required? What problem does it solve? -->
-<!-- If it resolves an open issue, please link to the issue here. For example "Resolves #1337" -->
+<!-- Describe your changes in detail. Why is this change required? What problem does it solve? -->
 
-### 📝 Checklist
+<!----------------------------------------------------------------------
+Before creating the pull request, please make sure you do the following:
 
-<!-- Put an `x` in all the boxes that apply. -->
-<!-- If your change requires a documentation PR, please link it appropriately -->
-<!-- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
+- Check that there isn't already a PR that solves the problem the same way. If you find a duplicate, please help us reviewing it.
+- Read the contribution docs at https://nuxt.com/docs/community/contribution
+- Read the Contributing Guidelines at https://github.com/vitejs/vite/blob/main/CONTRIBUTING.md.
```

Comment:
These two could probably be combined into one page

Reply:
but these are the vite contributing guidelines, right?

Reply:
Oh yeah, my mind just went on autopilot and thought to link pointed to a `CONTRIBUTING.md` in this repository. Awkward🙃